security/src/main/java/uk/ac/ox/ndph/mts/client/practitioner_service/PractitionerServiceClient.java

```diff
@@ -9,6 +9,6 @@
  */
 public interface PractitionerServiceClient {
 
-    List<RoleAssignmentDTO> getUserRoleAssignments(String userId);
+    List<RoleAssignmentDTO> getUserRoleAssignments(String userId, String token);
 
```

Comment:
Can the token be injected by Spring rather than as a parameter? We will have it everywhere if not.

Reply:
This is one of the commits from story 609 / PR 148, not this story.   Do you mind if we move this comment to https://github.com/NDPH-ARTS/mts-services/pull/148 so that @LizaShak can review it.  It will be confusing if we do it here.